elections.md

```diff
@@ -66,7 +66,8 @@ non-involved Steering Committee members will decide.
 
 ### Terms and Election Cycles
 
-Steering committee members are elected to serve two year terms.
+Steering committee members are elected to serve one, two year term. Members can
+serve two consecutive terms (4 years) and a lifetime of four terms (8 years).
```

Comment:
Sounds good as a whole. One question, what would happen if someone joined the SC due to a SC member stepping down, etc? 

For example, a SC member has a 1 year term left but they step down due to change in companies. Alice joins the SC as she had the next highest votes. Alice serves this 1 year term and decides to run again. She wins and serves the next 2 years. In total, she has now served for 3 years.

Will she be allowed to stand for the election again? If yes, would she need to step down after 1 year since her 4 years would be up?

Reply:
I'd say partial terms don't count. only full terms.